doc/BBCode.md

```diff
@@ -1,208 +1,611 @@
 Friendica BBCode tags reference
 ========================
 
-* [Home](help)
-
-Inline
------
-
-
-<pre>[b]bold[/b]</pre> : <strong>bold</strong>
-
-<pre>[i]italic[/i]</pre> : <em>italic</em>
```

Comment:
should not `<em>italic</em>` be a `<i>italic</i>` and `<strong>bold</strong>` be `<b>bold</b>`?

In most browser it is the same, but from the meaning of the tag, i is italic and em is emphasiszed, b is bold and strong is strongly emphasized.

I see this is how the converter does it, but then the converter needs to be fixed as well ;-) and maybe and `[em]` and a `[strong]¸ tag be included.

Just doing the German translation, so I could fix it in the English doc and the converter as well.